anime-flutter/anime_galaxy/lib/main.dart

```diff
@@ -39,6 +41,7 @@ class MyApp extends StatefulWidget {
   final LocalizationService _localizationService;
   final SwapThemeDataService _swapThemeService;
   final HomeModule _homeModule;
+  final SettingModule _settingModule;
```

Comment:
Why is this different from the boilerplate, please customize the boiler to fit your needs